analysers/analyser_merge_bicycle_parking_FR_lyon.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights XioNoX 2024                                                ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Merge import Analyser_Merge_Point, Source, GeoJSON, Load_XY, Conflate, Select, Mapping
+
+class Analyser_Merge_Bicycle_Parking_FR_Lyon(Analyser_Merge_Point):
+    def __init__(self, config, logger = None):
+        Analyser_Merge_Point.__init__(self, config, logger)
+        self.def_class_missing_official(item = 8150, id = 1, level = 3, tags = ['merge', 'public equipment', 'bicycle', 'fix:survey', 'fix:picture'],
+            title = T_('Bicycle parking not integrated'))
+        self.def_class_possible_merge(item = 8151, id = 3, level = 3, tags = ['merge'],
+            title = T_('Bicycle parking integration suggestion'))
+        self.def_class_update_official(item = 8152, id = 4, level = 3, tags = ['merge'],
+            title = T_('Bicycle parking update'))
+        self.init(
+            "https://www.data.gouv.fr/fr/datasets/parcs-de-stationnement-velos-de-la-metropole-de-lyon/",
+            "Localisation des stationnements vélos connus sur le territoire de la Métropole de Lyon",
+            GeoJSON(
+                Source(attribution = "Métropole de Lyon",
+                    fileUrl = "https://data.grandlyon.com/geoserver/metropole-de-lyon/ows?SERVICE=WFS&VERSION=2.0.0&request=GetFeature&typename=metropole-de-lyon:pvo_patrimoine_voirie.pvostationnementvelo&outputFormat=application/json&SRSNAME=EPSG:4326"),
+            ),
+            Load_XY("geom_x", "geom_y", where = lambda res: res["validite"] == "Validé"),
```

Comment:
Please, could you use a `select` in place of `where` here.

https://github.com/osm-fr/osmose-backend/blob/dev/doc/4-Merge.md#data-records

Reply:
Fixed, thanks.